<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Customer_engineer_dialog</class>
 <widget class="QDialog" name="Customer_engineer_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>298</width>
      <height>98</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Save picture:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Picture path:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Log path:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pp_edit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>130</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lp_edit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>130</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pp_btn">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lp_btn">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Original - all</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Original - not ok</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Screenshot - all</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Screenshot - not ok</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>470</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>O K</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>300</width>
     <height>350</height>
    </rect>
   </property>
   <property name="title">
    <string>Function options</string>
   </property>
   <widget class="QComboBox" name="unwrap_box_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>BRANCH_CUT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SPAN_BRANCH_CUT</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>psi:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>unwrap:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="psi_box_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>BUCKETB5A_H_P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BUCKET9A_CS_P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>OPT_SEQUENCE</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="filter_box_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>NONE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LOW_PASS</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>filter:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="r_s_no_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>remove_spikes:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="r_s_yes_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>refractive_index:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="r_r_yes_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="i_w_box">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>intf_scale_factor:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="a_t_itl_box_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="suffix">
     <string> ms</string>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="value">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="i_s_f_box">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>iso_wavelength:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>test_wavelength:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>remove_residual:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto interval:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="r_i_box">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="t_w_box">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="r_r_no_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No</string>
    </property>
   </widget>
   <widget class="QComboBox" name="unit_box">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>UNIT_TYPE_WAVE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UNIT_TYPE_FR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UNIT_TYPE_NM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UNIT_TYPE_UM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UNIT_TYPE_MM</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>unit:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
